content/learn/case-studies/hearst.md

```diff
@@ -0,0 +1,55 @@
+---
+$order: 7
+$title: Hearst
+$date: January 24, 2017
+headline: Hearst integrates key partner and product solutions on AMP
+class: case-study
+thumb: "case-studies/hearst_thumb.jpg"
+featured: "case-studies/hearst_framed.png"
+results:
+    - stat: 29%
+      description: Increase in viewability
+    - stat: 45%
+      description: Increase in CTR on Ads
+    - stat: +1M
+      description: AMP pages published
+logo:
+  src: "case-studies/hearst_logo.png"
+  width: 60
+  height: 14
+download: "case-studies/hearst.pdf"
+---
+
+<div class="img-right">
```

Comment:
nit: I'm not sure if the image s/b cropped at the top but if you look at the rendered page, the image doesn't look aligned with the section it's in and the image encroaches into the next section. It's not a major issue.